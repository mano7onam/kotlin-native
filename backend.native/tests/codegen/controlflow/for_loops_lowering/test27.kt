package codegen.controlflow.for_loops_lowering.test27

import kotlin.test.*

@Test fun runTest() {
	val arr = arrayOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
	for (i in 0 .. 10 step 2) {
		for (j in arr) {
			for (k in arrayOf("a", "b", "helolo world", "fff")) {
				println(j)
				println(k)
			}	    	
	    }
	}   
}
