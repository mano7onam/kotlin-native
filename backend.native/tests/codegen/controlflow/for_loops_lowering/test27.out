1
a
1
b
1
helolo world
1
fff
2
a
2
b
2
helolo world
2
fff
3
a
3
b
3
helolo world
3
fff
4
a
4
b
4
helolo world
4
fff
5
a
5
b
5
helolo world
5
fff
6
a
6
b
6
helolo world
6
fff
7
a
7
b
7
helolo world
7
fff
8
a
8
b
8
helolo world
8
fff
9
a
9
b
9
helolo world
9
fff
10
a
10
b
10
helolo world
10
fff
1
a
1
b
1
helolo world
1
fff
2
a
2
b
2
helolo world
2
fff
3
a
3
b
3
helolo world
3
fff
4
a
4
b
4
helolo world
4
fff
5
a
5
b
5
helolo world
5
fff
6
a
6
b
6
helolo world
6
fff
7
a
7
b
7
helolo world
7
fff
8
a
8
b
8
helolo world
8
fff
9
a
9
b
9
helolo world
9
fff
10
a
10
b
10
helolo world
10
fff
1
a
1
b
1
helolo world
1
fff
2
a
2
b
2
helolo world
2
fff
3
a
3
b
3
helolo world
3
fff
4
a
4
b
4
helolo world
4
fff
5
a
5
b
5
helolo world
5
fff
6
a
6
b
6
helolo world
6
fff
7
a
7
b
7
helolo world
7
fff
8
a
8
b
8
helolo world
8
fff
9
a
9
b
9
helolo world
9
fff
10
a
10
b
10
helolo world
10
fff
1
a
1
b
1
helolo world
1
fff
2
a
2
b
2
helolo world
2
fff
3
a
3
b
3
helolo world
3
fff
4
a
4
b
4
helolo world
4
fff
5
a
5
b
5
helolo world
5
fff
6
a
6
b
6
helolo world
6
fff
7
a
7
b
7
helolo world
7
fff
8
a
8
b
8
helolo world
8
fff
9
a
9
b
9
helolo world
9
fff
10
a
10
b
10
helolo world
10
fff
1
a
1
b
1
helolo world
1
fff
2
a
2
b
2
helolo world
2
fff
3
a
3
b
3
helolo world
3
fff
4
a
4
b
4
helolo world
4
fff
5
a
5
b
5
helolo world
5
fff
6
a
6
b
6
helolo world
6
fff
7
a
7
b
7
helolo world
7
fff
8
a
8
b
8
helolo world
8
fff
9
a
9
b
9
helolo world
9
fff
10
a
10
b
10
helolo world
10
fff
1
a
1
b
1
helolo world
1
fff
2
a
2
b
2
helolo world
2
fff
3
a
3
b
3
helolo world
3
fff
4
a
4
b
4
helolo world
4
fff
5
a
5
b
5
helolo world
5
fff
6
a
6
b
6
helolo world
6
fff
7
a
7
b
7
helolo world
7
fff
8
a
8
b
8
helolo world
8
fff
9
a
9
b
9
helolo world
9
fff
10
a
10
b
10
helolo world
10
fff
